### PR TITLE
Get-started-improvements

### PR DESCRIPTION
**Pull request by bmckercher123**
Improve topic descriptions, and move "Managed public cloud" section to second in TOC order.

- Merge to nightly

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit fe2e4afbdc41189ffffbfa189614c86b2ee79687

https://deploy-preview-3252--calico.netlify.com

File: _data/navbars/getting-started.yml [hunk 1]
```diff
@@ -26,7 +19,14 @@ section:
     - title: IBM Cloud Kubernetes Service (IKS)
       path: /getting-started/kubernetes/managed-public-cloud/iks
     - title: Microsoft Azure Kubernetes Service (AKS)
-      path: /getting-started/kubernetes/managed-public-cloud/aks
+      path: /getting-started/kubernetes/managed-public-cloud/aks
```
Comment (lxpollitt) [inline _data/navbars/getting-started.yml]: I think the order of pages should be: Quickstart, managed pub cloud, self-managed pub cloud, self-managed on-prem, flannel, hard way, system requirements.  

Note that we may change system requirements page to be earlier in the future, but for now I believe it likely causes more friction than it solves.

File: getting-started/kubernetes/hardway/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Up for the challenge? Calico the hard way takes you under the covers of a Calico installation.
+description: Go under the covers of an end-to-end Calico installation.
```
Comment (lxpollitt) [inline getting-started/kubernetes/hardway/index.md]: I quite liked the quirkiness of the previous text. I would tweak it to say "an end-to-end" though as you've suggested.

File: getting-started/kubernetes/self-managed-onprem/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Self-managed on-premises
-description: Install Calico networking and network policy for on-premises deployments. 
+description: Managing Calico on-premises provides the most flexibility and customization, but requires more resources and infrastructure.
```
Comment (lxpollitt) [inline getting-started/kubernetes/self-managed-onprem/index.md]: This change seems to go in the opposite direction to the others.  i.e. It takes it away from telling you what this section covers, and introduces doubt as to whether you should run Calico on-prem.  I suggest reverting to the previous text for this one.

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Install Calico networking and network policy for on-premises deployments
-description: Install Calico networking and network policy for on-premises deployments.
+description: Determine your datastore and use Calico manifest to customize the resources on each node in the cluster.
```
Comment (lxpollitt) [inline getting-started/kubernetes/self-managed-onprem/onpremises.md]: Suggest revert to the original text.  (Determining your datastore and using Calico manifests are steps in the how-to guide; they aren't a description of the how-to guide's goal.)

The Customize Manifest description is worth updating though (getting-started/kubernetes/installation/config-options.md):
Customize manifests prior to installing Calico (recommended) -> Optionally customize manifests prior to installing Calico.

File: getting-started/kubernetes/self-managed-public-cloud/index.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Self-managed public cloud
-description: Manage your own Kubernetes clusters in a public cloud provider.
+description: Manage your own Kubernetes clusters in AWS, GCE, or Azure public cloud provider.
```
Comment (lxpollitt) [inline getting-started/kubernetes/self-managed-public-cloud/index.md]: public cloud provider -> public clouds.

File: getting-started/kubernetes/self-managed-onprem/index.md (hunk 3 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/index.md]: But the original text (Install Calico networking and network policy for on-premises deployments), was the same for the topic as the landing page. I understood you did not want it repeated, so I tried to come up with something different.

File: getting-started/kubernetes/self-managed-public-cloud/index.md (hunk 5 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-public-cloud/index.md]: Fixed

File: getting-started/kubernetes/self-managed-onprem/onpremises.md (hunk 4 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: Fixed

File: getting-started/kubernetes/hardway/index.md (hunk 2 above)
Comment (bmckercher123) [reply getting-started/kubernetes/hardway/index.md]: Fixed.

File: _data/navbars/getting-started.yml (hunk 1 above)
Comment (bmckercher123) [reply _data/navbars/getting-started.yml]: Fixed.

File: getting-started/kubernetes/self-managed-onprem/index.md (hunk 3 above)
Comment (lxpollitt) [reply getting-started/kubernetes/self-managed-onprem/index.md]: Must have been a miscommunication on my part, but alas I don't recall what I may have meant to say. I suggest reverting to the previous text for this one.

**Pull request merged**